examples/planning/add_keyword_plan.rb

```diff
@@ -38,8 +38,8 @@ def add_keyword_plan(customer_id)
     customer_id,
     plan_campaign,
   )
-  create_keyword_plan_keywords(client, customer_id, plan_ad_group)
-  create_keyword_plan_negative_keywords(client, customer_id, plan_campaign)
+  create_keyword_plan_adgroup_keywords(client, customer_id, plan_ad_group)
```

Comment:
should this be "create_keyword_plan_ad_group_keywords"?

see line 36

Reply:
done.